### PR TITLE
Coconet: Remove conv weights initializer when not training...

### PR DESCRIPTION
**Pull request by jhowcrof**
(was causing TPU errors and isn't needed for inference) and remove tf.nn.moments from batchnorm when not training (was a bug, shouldn't have been in inference graph).

### CODE REVIEW

**Comment by jhowcrof** [comment]
thanks!
